specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -224,24 +236,36 @@ FPVM state to track memory reservations initiated by `ll` operations.
 
 When an `ll` instruction is executed:
 
-- `llReservationActive` is set to true.
+- `llReservationStatus` is set to `1`.
 - `llAddress` is set to the memory address specified by `ll`.
 - `llOwnerThread` is set to the `threadID` of the active thread.
 
 Only a single memory reservation can be active at a given time - a new reservation will clear any previous reservation.
 
 When the VM writes any data to memory, these `ll`-related fields are checked and the memory reservation is cleared if
-a memory write touches a reserved `llAddress`.
+a memory write touches a `Word` that contains `llAddress`.
 
 When an `sc` instruction is executed, the operation will only succeed if:
 
-- There exists an active reservation (`llReservationActive == true`).
+- There exists an active 4-byte memory reservation (`llReservationStatus == 1`).
```

Comment:
Could you clarify this better.

This line can be parsed incorrectly in several ways. I originally read the 4-byte memory reservation as the width of the reserved memory location. [Which isn't the case](https://github.com/ethereum-optimism/optimism/blob/develop/cannon/mipsevm/multithreaded/mips.go#L331C1-L334C3).
My second read was that it referred to the 4-byte memory reservation as the width of `llReservationStatus`, which isn't correct either.

Upon my third read, I finally inferred that the line describes the bit length of the operation used to create the reservation (i.e. `ld`). Can we be more explicit what this means. The phrase "4-byte memory reservation" hasn't been defined prior to this so it'll be worthwhile to clarify what that means. You could also simply replace this line with:
> When `llReservationStatus` is `1`

to avoid any ambiguity on what "4-byte memory" means in this context.

Reply:
👍 simplified this